manifests/init.pp

```diff
@@ -120,6 +136,11 @@
     amqp_truststore              => $::certs::candlepin::amqp_truststore,
     qpid_ssl_cert                => $::certs::qpid::client_cert,
     qpid_ssl_key                 => $::certs::qpid::client_key,
+    db_host                      => $katello::candlepin_db_host,
```

Comment:
What do you think about getting these from `$::candlepin::db_*` params ? I think it should mean we wouldn't have to set defaults here and in puppet-candlepin.